### ISSUE TITLE
Return to Dust only causes Ranar the Ever-Watchful to trigger once instead of twice.

### ISSUE THREAD

**Issue by Tharkon** [magefree/mage#11451]
When casting [[Return to Dust]] during your main phase and exile two different targets, it causes [[Ranar the Ever-Watchful]] to trigger once instead of twice.

Since the word exile appears twice on the card, these are two separate instances of a card being exiled rather than one instance of two cards being exiled.

[[Settle Beyond Reality]] does cause Ranar to trigger correctly, so perhaps the code could be changed to be similar to that.
From what little experience I have reading the code, I think it matters than Settle Beyond Reality has two effects while Return to Dust has a single effect. It's not modal though, so maybe more like Opt.

**Comment by jeffwadsworth** [comment]
To my knowledge, Ranar the Ever-Watchful's ability is "one or more", so if you exile 1 or 50, it fires once.  Do you have some judge's decision on this one?

**Comment by jeffwadsworth** [comment]
<html><body>

6/18/2021 | Ranar's  triggered ability triggers only once for each time you exile cards from  your hand or permanents from the battlefield, no matter how many  objects were exiled at the same time.
-- | --



</body>
</html>

**Comment by Tharkon** [comment]
Ranar triggers once for each time you exile one or more permanents. 
Return to Dust does not exile two target permanents, it exiles target permanent and then it exiles another target permanent.
Each time a verb is used is a separate action.

If you want a judge's opinion, there's this: https://www.youtube.com/watch?v=eDp8Bozl4lE

**Comment by jeffwadsworth** [comment]
Return to Dust must be coded to allow for two separate exile instances.

**Comment by xenohedron** [comment]
1. Should use `AddendumCondition` rather than making a new condition- this correctly handles the case of copying if it wasn't cast.
2. Rather than both permanents under a single Target, could use two separate Targets (first required, second optional), this enables a standard ExileTargetEffect for the first, and the second effect just uses SecondTargetPointer (could also be standard except for that "you may", so I guess still has to be custom)

Rulings for reference:
Return to Dust can always target a second artifact or enchantment; it just won't exile it if it's not your main phase when you cast Return to Dust. (2021-03-19)
If a spell or ability copies Return to Dust, the copy exiles only the first target artifact or enchantment. This is because the copy wasn't cast at all. (2021-03-19)

also just curious, is there a reason for Outcome.Detriment rather than Outcome.Exile?

**Comment by jeffwadsworth** [comment]
I will test the copy issue.  The addendum in the condition name makes it hard to follow and find.
Easiest just to target one or both.
The outcome is used for AI and makes it clear.  The chance of it exiling its own permanents being beneficial is quite unlikely.
